Code/Framework/AzCore/AzCore/Math/Sphere.h

```diff
@@ -18,12 +18,12 @@ namespace AZ
     class Sphere
     {
     public:
-
         AZ_TYPE_INFO(Sphere, "{34BB6527-81AE-4854-99ED-D1A319DCD0A9}");
 
         Sphere() = default;
 
         explicit Sphere(const Vector3& center, float radius);
+        Sphere(const Sphere& sphere);
```

Comment:
update: We can delete `explicit` while we're here and also adding the copy constructor for a type like this is actually redundant, the compiler will just generate this unless we explicitly remove it, so we should try and follow the rule of 0 (or rule of 5 for more complex types which manage their own resources etc...)